frame/contracts/src/wasm/runtime.rs

```diff
@@ -1043,10 +1083,51 @@ define_env!(Env, <E: Ext>,
 		output_len_ptr: u32
 	) -> ReturnCode => {
 		ctx.call(
-			CallFlags::from_bits(flags).ok_or_else(|| "used reserved bit in CallFlags")?,
-			callee_ptr,
-			gas,
-			value_ptr,
+			CallFlags::from_bits(flags).ok_or_else(|| Error::<E::T>::InvalidCallFlags)?,
+			CallType::Call{callee_ptr, value_ptr, gas},
+			input_data_ptr,
+			input_data_len,
+			output_ptr,
+			output_len_ptr,
+		)
+	},
+
+	// Execute code in the context of the current contract.
```

Comment:
```suggestion
	// Execute code in the context of the current contract.
	// 
	// In this case, context refers to the current contract's: storage, caller, and value.
```

It would be nice to get those three linked to their docs if possible